rpc/client/client.go

```diff
@@ -751,13 +748,13 @@ func (c *Client) Status(ctx context.Context) (*ctypes.ResultStatus, error) {
 			LatestAppHash:     latestAppHash[:],
 			LatestBlockHeight: int64(latestHeight),
 			LatestBlockTime:   time.Unix(0, int64(latestBlockTimeNano)),
+			CatchingUp:        c.node.BlockManager.TargetHeight.Load() > latestHeight,
```

Comment:
u need to set `TargetHeight` also in  `syncBlockManager` , currnetly u made it event based only

Reply:
this field needs a docstring as well IMO
the other ones are obvious and fine

Reply:
fix pushed